rpm/update-spec-provides.sh

```diff
@@ -4,12 +4,27 @@
 # packaging, via the `propose-downstream` packit action.
 # The goimports don't need to be present upstream.
 
-set -eo pipefail
+set -eox pipefail
 
 PACKAGE=skopeo
 # script is run from git root directory
 SPEC_FILE=rpm/$PACKAGE.spec
 
+# Needed for golist to work
+# This is run in the packit sandbox environment for downstream tasks and
+# doesn't affect upstream repo and CI.
+GOPATHDIR=~/go/src/github.com/containers/
+mkdir -p $GOPATHDIR
+ln -sf $(pwd) $GOPATHDIR/.
+
+# Packit sandbox doesn't allow root
+# Install golist by downloading and extracting rpm
+# We could handle this in packit `sandcastle` upstream itself
+# but that depends on golist existing in epel
+# https://github.com/packit/sandcastle/pull/186
+dnf download golist
+rpm2cpio golist-*.rpm | cpio -idmv
+
 sed -i '/Provides: bundled(golang.*/d' $SPEC_FILE
 
 GO_IMPORTS=$(golist --imported --package-path github.com/containers/$PACKAGE --skip-self | sort -u | xargs "-I{}" echo "Provides: bundled(golang({}))")
```

Comment:
(BTW looking at what Golist does, it seems that it should be possible to use the standard Go command, something like `go list -deps -f '{{range .Imports}}{{.}}\n{{end}}'  ./...`. I fully support using the same command that `go-rpm-macros` uses, and I know nothing about the environment — quite possibly installing all of Go is even worse than installing `golist` — I just wanted to note the theoretical possibility, now that I took some time to learn what `golist` is.)

Reply:
ack. I haven't checked with packit upstream about this, but golang might be a heavy dependency to include in their sandbox. So, I'll keep golist for now.